tests/unit-core/test-api.c

```diff
@@ -895,7 +901,10 @@ main (void)
     char check_value[] = "{\"name\":\"John\"}";
     jerry_value_t key = jerry_create_string ((const jerry_char_t *) "name");
     jerry_value_t value = jerry_create_string ((const jerry_char_t *) "John");
-    jerry_set_property (obj, key, value);
+    res = jerry_set_property (obj, key, value);
+    TEST_ASSERT (!jerry_value_is_error (res));
+    TEST_ASSERT (jerry_get_boolean_value (res));
```

Comment:
Please add a `jerry_get_boolean_value` call too to ensure the result was a success. 
```c
TEST_ASSERT (jerry_get_boolean_value (res) && jerry_get_boolean_value (res));
```